packages/story-editor/src/output/page.js

```diff
@@ -130,6 +138,28 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
           </a>
         </amp-story-page-outlink>
       )}
+      {videoCaptions.length > 0 && (
+        <amp-story-grid-layer
+          template="vertical"
+          aspect-ratio={ASPECT_RATIO}
+          class="grid-layer"
+        >
+          <div className="page-fullbleed-area">
```

Comment:
Does this guarantee that captions are rendered within the safe zone? Do we need to  nest `<div className="page-safe-area">`?

Reply:
Haven't tested this in depth yesterday but yeah I think it needs `page-safe-area` as well, but that will obviously require some styling changes since the basic scrim I applied here won't work anymore:

![Screenshot 2021-09-16 at 21 51 27](https://user-images.githubusercontent.com/841956/133676990-c908f5b1-50d5-4280-bae4-9e46a30cebd6.png)
![Screenshot 2021-09-16 at 21 50 42](https://user-images.githubusercontent.com/841956/133676996-a85bf024-c6bd-4505-84b1-3248184bca4f.png)